omod/src/main/resources/apps/isanteplus_registration_app.json

```diff
@@ -142,6 +142,16 @@
                                         "providerName": "registrationapp",
                                         "fragmentId": "field/fingerprintM2sys"
                                     }
+                                },
+                                {
+                                    "type": "nationalPatientIdentifier",
```

Comment:
Is this correct? won't this result in the widget for scanning being shown twice on the page?

Reply:
oops, yes I will undo it